### PR TITLE
Office Supply Map Updates

### PR DESCRIPTION
**Pull request by Apogee-dev**
Updates the maps with new pens and office supplies. Except for Defficiency, because I can't load that without crashing.

:cl:
 * rscadd: Added color-coded paper bins and folders to all maps except Deff.
 * rscadd: Added fountain pens and multipens to all maps except Deff.

### CODE REVIEW

**Comment by Kurfursten** [comment]
Waiting for #30684 to clear

**Comment by Kurfursten** [comment]
Shifty said to go ahead of his

File: vgstation13.dme [hunk 1]
```diff
@@ -2757,7 +2757,7 @@
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
 #include "maps\_map_override.dm"
-#include "maps\tgstation.dm"
+#include "maps\lowfatbagel.dm"
```
Comment (Kurfursten) [inline vgstation13.dme:2760]: oh no no no no no

Comment (Apogee-dev) [reply vgstation13.dme:2760]: Accidentally left that in from testing. Oops.

**Pull request merged**